### PR TITLE
IPublishedContent IsHelpers for v8

### PR DESCRIPTION
**Pull request by poornimanayar**
I have updated the IsHelper for `IPublishedContent` as requested in #1875.  I have renamed the existing documentation for `v7` and made a new one for `v8`.  Please let me know what you think :-)

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks @poornimanayar ! 😄 I'll make sure this is reviewed as soon as possible.

File: Reference/Querying/IPublishedContent/IsHelpers/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## IsHelper Methods
+
+### IsComposedOf(string alias)
+
+Test whether the content is of a content type composed of the given alias
+
+### IsAllowedTemplate(int templateId)
+
+Test whether the specified templateId is an allowed template for the current node
```
Comment (sofietoft) [inline Reference/Querying/IPublishedContent/IsHelpers/index.md]:
```suggestion
Test whether the specified `templateId` is an allowed template for the current node 
```

File: Reference/Querying/IPublishedContent/IsHelpers/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Test whether the content is of a content type composed of the given alias
+
+### IsAllowedTemplate(int templateId)
+
+Test whether the specified templateId is an allowed template for the current node 
+
+### IsAllowedTemplate(string templateAlias)
+
+Test whether the specified templateAlias is an allowed template for the current node
```
Comment (sofietoft) [inline Reference/Querying/IPublishedContent/IsHelpers/index.md]:
```suggestion
Test whether the specified `templateAlias` is an allowed template for the current node 
```

**Comment by poornimanayar** [comment]
Hi @sofietoft ,

I have made the amends as per your comments. I have added the _dot_ to the v7 version as well.

Poornima

**Comment by sofietoft** [comment]
Brilliant! Thanks for putting in the effort to do all this 😄 🎉

**Pull request merged**